src/main/java/io/debezium/connector/spanner/db/mapper/ChangeStreamRecordMapper.java

```diff
@@ -134,12 +257,145 @@ DataChangeEvent toDataChangeEvent(Partition partition, Struct row,
                 streamEventMetadataFrom(partition, commitTimestamp, resultSetMetadata));
```

Comment:
Do we want to use modTypeFrom and valueCaptureTypeFrom here?

Reply:
Done